test/spoom/cli/bump_test.rb

```diff
@@ -536,6 +537,70 @@ class A; end
           class A; end
         RB
       end
+
+      def test_multiple_constant_definitions
+        @project.write("file1.rb", <<~CONTENTS)
+          # typed: ignore
+          class Bar; end
+          Foo = Bar
+        CONTENTS
+
+        @project.write("file2.rb", <<~CONTENTS)
+          # typed: false
+          class Foo; end
+        CONTENTS
+
+        result = @project.bundle_exec("spoom bump --no-color --from ignore --to false")
+        assert_empty(result.err)
+        assert_equal(<<~OUT, result.out)
+          Checking files...
+
+          No file to bump from `ignore` to `false`
+        OUT
+        assert(result.status)
+      end
+
+      # This test currently fails and represents a bug in Spoom.
+      # The bug here is that I have a base class that is ignored, so Sorbet reads the RBI for that class instead
+      # When that parent class is typed, Sorbet no longer looks at that RBI,
+      # so it no longer knows about the `foo` method.
+      # However, `file1.rb` is not included in the error message, and it probably couldn't,
+      # because theoretically the method could come from anywhere,
+      # so Sorbet probably wouldn't want to list all of the places
+      # a method COULD come from and say none of them have the method.
+      #
+      # def test_untyped_base_class_errors
+      #   @project.write("file1.rb", <<~CONTENTS)
+      #     # typed: ignore
+      #     ParentClass = 'my_dynamically_fetched_parent_class'
+      #     class MyBaseClass < ParentClass; end
```

Comment:
If you change that `sorbet.run` to `typed: false` that error goes away.

Reply:
Indeed, yet I still do not understand this test-case:

```
$ cat file1.rb
# typed: false

ParentClass = 'my_dynamically_fetched_parent_class'
class MyBaseClass < ParentClass; end

$ cat file2.rbi
# typed: true

class MyBaseClass
  def foo; end
end

$ cat file3.rb
# typed: true

class MyChildClass < MyBaseClass
  def some_method
    foo
  end
end

$ bundle exec srb tc --no-config file1.rb file2.rbi file3.rb
No errors! Great job.
```

What are we expecting here?

Reply:
I think there should be an error here – but you're right, my test case didn't properly reproduce it, that's my bad.

Here's what I'm expecting...

1) When `file1.rb` is `typed: ignore`, it uses `file2.rbi` to understand that `MyBassClass` gives `MyChildClass` a `foo` method.
2) When `file1.rb` is turned to `typed: false`, I expect an error because it stops looking at the RBI.

That's not happening in this test case, but is happening in our codebase, I think because there is something more complex going on. Let me keep digging in.